### PR TITLE
improve cutting

### PR DESCRIPTION
**Pull request by teolemon**
https://static.openfoodfacts.org/images/products/351/269/000/4987/ingredients_fr.10.full.jpg

```
farine de _BLE_, eau, huiles et graisses végétales (palme, colza), sucre, sel, jus de citron concentré, levure désactivée, conservateur : sorbate de potassium. Produit
``` 
to 
```
farine de _BLE_, eau, huiles et graisses végétales (palme, colza), sucre, sel, jus de citron concentré, levure désactivée, conservateur : sorbate de potassium. 
```

### CODE REVIEW

File: lib/ProductOpener/Ingredients.pm [hunk 1]
```diff
@@ -2327,6 +2327,7 @@ fr => [
 'conservation:',
 '(a|à) protéger de ', # humidité, chaleur, lumière etc.
 'conditionn(e|é) sous atmosph(e|è)re protectrice',
```
Comment (stephanegigandet) [inline lib/ProductOpener/Ingredients.pm:2329]:
```suggestion
'(produit )?conditionn(e|é) sous atmosph(e|è)re protectrice',
```

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -2327,6 +2327,7 @@ fr => [
 'conservation:',
 '(a|à) protéger de ', # humidité, chaleur, lumière etc.
```
Comment (stephanegigandet) [inline lib/ProductOpener/Ingredients.pm:2328]:
```suggestion
'(produit )?(a|à) protéger de ', # humidité, chaleur, lumière etc.
```

File: lib/ProductOpener/Ingredients.pm [hunk 3]
```diff
@@ -2327,6 +2327,7 @@ fr => [
...
 'conditionn(e|é) sous atmosph(e|è)re protectrice',
+'produit conditionn(e|é) sous atmosph(e|è)re protectrice',
```
Comment (stephanegigandet) [inline lib/ProductOpener/Ingredients.pm:2330]: add (produit )? in regexp above instead

File: lib/ProductOpener/Ingredients.pm [hunk 4]
```diff
@@ -2325,8 +2325,9 @@ fr => [
 '(conditions|conseils) de conservation',
 'conseil d\'utilisation',
 'conservation:',
-'(a|à) protéger de ', # humidité, chaleur, lumière etc.
-'conditionn(e|é) sous atmosph(e|è)re protectrice',
+'(produit )?(a|à) protéger de ', # humidité, chaleur, lumière etc.
+'(produit )?conditionn(e|é) sous atmosph(e|è)re protectrice',
+'produit conditionn(e|é) sous atmosph(e|è)re protectrice',
```
Comment (teolemon) [inline lib/ProductOpener/Ingredients.pm:2330]:
```suggestion
```

File: lib/ProductOpener/Ingredients.pm (hunk 3 above)
Comment (teolemon) [reply lib/ProductOpener/Ingredients.pm]: fixed

**Pull request merged**